pom.xml

```diff
@@ -165,6 +166,49 @@
             </plugins>
         </pluginManagement>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>3.7.1</version>
```

Comment:
If we go with this solution, I would move the plugin version on the `</pluginManagement>` and execute the plugin only on the core pom